src/main/java/org/jberet/jpa/repository/JpaRepository.java

```diff
@@ -64,7 +64,7 @@ public final class JpaRepository implements JobRepository {
 
     private final Map<String, Object> hints = new HashMap<String, Object>() {
         {
-            put("javax.persistence.lock.timeout", -2);
+            put("jakarta.persistence.lock.timeout", -2);
```

Comment:
there is another occurrence in line 344. Can you check if it should also be replaced?

Reply:
Sorry, you are right
I created a constant JAKARTA_JPA_LOCK_TIMEOUT and used that on both cases